example/packing_characterization_run.py

```diff
@@ -0,0 +1,301 @@
+#/////////////////////////////////////////////////
+__author__      = "Chengshun Shang (CIMNE)"
+__copyright__   = "Copyright (C) 2023-present by Chengshun Shang"
+__version__     = "0.0.1"
+__maintainer__  = "Chengshun Shang"
+__email__       = "cshang@cimne.upc.edu"
+__status__      = "development"
+__date__        = "June 21, 2024"
+__license__     = "BSD 2-Clause License"
+#/////////////////////////////////////////////////
+
+import KratosMultiphysics
+from KratosMultiphysics import *
+from KratosMultiphysics.DEMApplication import *
+from KratosMultiphysics.DEMApplication.DEM_analysis_stage import DEMAnalysisStage
+from KratosMultiphysics.DEMApplication import DEM_procedures as DEM_procedures
+
+import matplotlib.pyplot as plt
+import json
+
+class ParticlePackingCharacterizationRun(DEMAnalysisStage):
```

Comment:
I wonder why this Python file is here. Should it be located on "./src/data_processing/post_processing/"?

Reply:
I think is in both sides. When you run a case the code copies some files from src folder

Reply:
Yes, we always need to copy a file from the src folder, so we do not need to put a script file here. The "example" folder is only used to store some example cases. The user cases could be somewhere else. So, I suggest not putting a script file in this folder. What do you think?